src/test/kotlin/no/nav/tilleggsstonader/sak/brev/brevmottaker/BrevmottakereServiceTest.kt

```diff
@@ -0,0 +1,318 @@
+package no.nav.tilleggsstonader.sak.brev.brevmottaker
+
+import io.mockk.every
+import io.mockk.mockk
+import no.nav.tilleggsstonader.kontrakter.brevmottaker.BrevmottakerOrganisasjonDto
+import no.nav.tilleggsstonader.kontrakter.brevmottaker.BrevmottakerPersonDto
+import no.nav.tilleggsstonader.kontrakter.brevmottaker.MottakerRolle
+import no.nav.tilleggsstonader.libs.test.assertions.catchThrowableOfType
+import no.nav.tilleggsstonader.sak.behandling.BehandlingService
+import no.nav.tilleggsstonader.sak.behandling.domain.Behandling
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingKategori
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingResultat
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingStatus
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingType
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingÅrsak
+import no.nav.tilleggsstonader.sak.behandlingsflyt.StegType
+import no.nav.tilleggsstonader.sak.infrastruktur.database.Sporbar
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.ApiFeil
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.http.HttpStatus.BAD_REQUEST
+import java.util.*
+
+class BrevmottakereServiceTest {
+
+    private val brevmottakereRepository = mockk<BrevmottakerRepository>()
+    private val behandlingService = mockk<BehandlingService>()
+
+    private val brevmottakereService: BrevmottakereService =
+        BrevmottakereService(brevmottakereRepository, behandlingService)
+
+    @Test
+    fun `lagreBrevmottakere skal kaste feilmelding dersom behandligsstatus er FATTER_VEDTAK`() {
+        val brevmottakerPersonDto =
+            BrevmottakerPersonDto(UUID.randomUUID(), "12312312323", "Test Testersen", MottakerRolle.BRUKER)
+        val brevmottakerOrganisasjonDto =
+            BrevmottakerOrganisasjonDto(UUID.randomUUID(), "45645646456", "Forretninger AS", "Navn navnesen")
+        val brevmottakereDto =
+            BrevmottakereDto(listOf(brevmottakerPersonDto), listOf(brevmottakerOrganisasjonDto))
+
+        val behandling = Behandling(
```

Comment:
Det finnes test-util-metoder for dette, se `fun behandling(` i `DomainUtil`

Reply:
Her skjønner jeg ikke hva du mener? Sier du at denne funksjonaliteten er testet andre steder i koden? Eller at testene kan skrives på en "smoothere" måte?

Reply:
Vi fikser det nå!